NtupleProducer/python/runPerformanceNTuple.py

```diff
@@ -290,6 +290,31 @@ def addAllJets():
     addCaloJets()
     #addTkJets()
 
+
+def addL1Tracks():
+    process.l1trackTable = cms.EDProducer("L1TrackTableProducer",
+                                          tracks = cms.InputTag("l1tTTTracksFromTrackletEmulation", "Level1TTTracks"),
+                                          trackingParticleMap = cms.InputTag("TTTrackAssociatorFromPixelDigis", "Level1TTTracks"),
+                                          selection = cms.string("momentum().perp()>0"),
+                                          name= cms.string("L1Tk"),
+                                          variables = cms.PSet(
+                                             chi2 = cms.string("chi2"),
+                                             chi2Red = cms.string("chi2Red"),
+                                             #chi2Bend = cms.string("chi2Bend"),
+                                             #chi2BendRed = cms.string("chi2BendRed"),
+                                             chi2XYRed = cms.string("chi2XYRed"),
+                                             chi2ZRed = cms.string("chi2ZRed"),
+                                             d0 = cms.string("d0"),
+                                             nFitPars = cms.string("nFitPars"),
+                                             stubPtConsistency = cms.string("stubPtConsistency"),
+                                             z0 = cms.string("z0"),
+                                             nStub = cms.string("getStubRefs.size"),
+                                             hitPattern = cms.string("hitPattern"),
+                                             trkMVA1 = cms.string("trkMVA1")
+
+                                           ),
+                                       )
+
 def addJetConstituents(N):
     for i in range(N): # save a max of N daughters (unfortunately 2D arrays are not yet supported in the NanoAOD output module)
```

Comment:
Note this PR introduced this functionality into CMSSW_15_X: https://github.com/cms-sw/cmssw/pull/46702
(but it was developed in 14_2 so might also work out here)